elide-core/src/test/java/example/FosterParent.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2016, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package example;
+
+import com.yahoo.elide.annotation.Include;
+import com.yahoo.elide.annotation.ReadPermission;
+import com.yahoo.elide.security.checks.prefab.Role;
+import lombok.ToString;
+
+import javax.persistence.Entity;
+import java.util.Set;
+
+@Include(rootLevel = true, type = "foster", inheritPermissions = false)
+@Entity
+@ToString
+public class FosterParent extends Parent {
```

Comment:
I seem to have a Hibernate error if I include this FosterParent class in the integration tests.